src/plugins/nginx.go

```diff
@@ -352,6 +396,22 @@ func (n *Nginx) applyConfig(cmd *proto.Command, cfg *proto.Command_NginxConfig)
 	}
 }
 
+func isNapInPayload(directoryMap *proto.DirectoryMap, action proto.NginxConfigAction, path string) bool {
+	if (action == proto.NginxConfigAction_APPLY && directoryMap != &proto.DirectoryMap{}) {
+		for _, directory := range directoryMap.Directories {
+			log.Tracef("directory %v", directory.Name)
```

Comment:
Can you add more details to this trace log? 
Something like
`checking if nap metadata file is in directory %s`